tutorials/v7/draw_mt.cxx

```diff
@@ -53,25 +57,43 @@ void draw_canvas(const std::string &title, RColor col)
    }
 
    // Create a canvas to be displayed.
-   auto canvas = RCanvas::Create(title);
+   auto canvas = RCanvas::Create(title + " canvas");
    canvas->Draw(pHist)->SetLineColor(col);
+   // canvas->Draw(pHist)->SetLineColor(RColor::kRed);
    canvas->Draw(pHist2)->SetLineColor(RColor::kBlue);
 
-   canvas->Show();
 
-   for (int loop=0;loop<50;++loop) {
+   bool batch = gROOT->IsWebDisplayBatch();
```

Comment:
I find it surprising that `root -b draw_mt.cxx` opens windows, even if those are browser windows (it did for me, with master at least before this PR). Is this intended?

Reply:
You should call:

    root -b --web chromium draw_mt.cxx

This is a way now to configure batch mode for web display